backend/src/database/models.rs

```diff
@@ -390,16 +389,37 @@ pub struct Role {
     pub updated_at: NaiveDateTime,
 }
 
-#[derive(Insertable)]
+#[derive(FromForm)]
+pub struct RoleUpdateInput {
```

Comment:
Is there a reason this has to be a new struct? Can you not `#[derive(FromForm]` on struct `NewOrUpdateRole`?

Reply:
Thanks, I think I was tired when writing this, should be a bit cleaner now